tools/src/Transforms.ts

```diff
@@ -41,14 +41,18 @@ export function transformString(
 
 async function getTransformedFileContentAsync(
   filePath: string,
-  transforms: FileTransform[]
+  transforms: FileTransform[],
+  unusedTransforms?: Set<FileTransform>
```

Comment:
If the function name starts from `get`, it definitely shouldn't mutate any argument 😅
Maybe we can return `{ content: string, unusedTransforms: Set<FileTransform> }` instead and rename to something like `applyTransformsOnFileContentAsync`?
`copyFilesWithTransformsAsync` already returns an object with `content` so the result type can be partially shared.